### PR TITLE
adds a new fermentable drink meant for ashwalkers to create

### PR DESCRIPTION
**Pull request by boodaliboo**
this pr adds cactus cooler a decently alcoholic drink made from fermenting cactus it also heals a minor amount of burn damage hopefully encouraging the use of lavaland fermenting
#### Changelog

:cl:  
rscadd: Added a new alcoholic drink thats fermentable from lavaland cacti
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
ill fix that error when i come back

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -2092,3 +2092,20 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_icon_state = "planet_cracker"
 	glass_name = "Planet Cracker"
 	glass_desc = "Although historians believe the drink was originally created to commemorate the end of an important conflict in man's past, its origins have largely been forgotten and it is today seen more as a general symbol of human supremacy."
+	
+/datum/reagent/consumable/ethanol/cactuscooler
+	name = "Cactus Cooler"
+	description = "an alcoholic drink created by fermenting cactus, its color is odd looking"
```
Comment (Ktlwjec1) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: "**A**n alcoholic drink created by fermenting cactus, its color is odd looking**.**"

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 2]
```diff
@@ -2092,3 +2092,20 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	color = "#78b477" // rgb: 120, 180, 119
+	nutriment_factor = 2 * REAGENTS_METABOLISM
+	boozepwr = 25
+	taste_description = "refreshing and cooling"
+	glass_icon_state = "glass_green"
+	glass_name = "glass of cactus cooler"
+	glass_desc = "The byproduct of fermenting a cactus. For those wanting a refreshing drink in a barren wasteland"
```
Comment (Ktlwjec1) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: wasteland**.**

**Comment by boodaliboo** [comment]
WHY DID IT FAIL I LITTERALLY ONLY FIXED SPELLING ERRORS

**Comment by boodaliboo** [comment]
thanks ktl

**Review by JamieD1 — APPROVED** [review]

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 3]
```diff
@@ -2092,3 +2092,20 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	boozepwr = 25
+	taste_description = "refreshing and cooling"
+	glass_icon_state = "glass_green"
+	glass_name = "glass of cactus cooler"
+	glass_desc = "The byproduct of fermenting a cactus. For those wanting a refreshing drink in a barren wasteland."
+
+/datum/reagent/consumable/ethanol/cactuscooler/on_mob_life(mob/living/carbon/M)
+	if(M.getFireLoss() && prob(10))
+		M.heal_bodypart_damage(1)
+		. = 1
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]:
```suggestion
	if(M.getFireLoss() && prob(10))
		M.heal_bodypart_damage(0, 1)
		. = 1
```
The first argument is bruteloss, the second one is fireloss

**Review by alexkar598 — CHANGES_REQUESTED** [review]

**Comment by boodaliboo** [comment]
i think i commited them right

**Comment by Marmio64** [comment]
should make a new PR to modify this to heal a minor bit of brute and burn since eating cactus does that on its own

**Pull request merged**